src/main/java/com/ft/methodecontentcollectionmapper/model/ContentCollectionType.java

```diff
@@ -23,7 +23,7 @@ public String getCorrespondingContentUriSuffix() {
 
   public static ContentCollectionType fromWebType(String webType, String uuid)
       throws XPathExpressionException {
-    if (webTypeToContentCollectionTypeMap.get(webType) != null) {
+    if (webTypeToContentCollectionTypeMap.get(webType) == null) {
```

Comment:
This could be slightly optimized :)

Get the value from the map and put it in a variable. If the variable is null, throw exception, otherwise return the variable. No need to "double get".